### ISSUE TITLE
Notifications: Cha-Ching push notification sound disabled for free orders

### ISSUE THREAD

**Issue by seb86** [woocommerce/woocommerce-android#2632]
**Describe the solution you'd like**
Either disabled by default or the option to disable the "Cha-Ching" notification sound for orders that are free.

**Comment by seb86** [comment]
Any updates at all on this one?

**Comment by AmandaRiu** [comment]
Hi there! Please submit feature request to our [feature request portal](https://ideas.woocommerce.com/forums/133476-woocommerce?category_id=84283). Thanks!

**Comment by designsimply** [comment]
I checked the ideas portal for [notifications](https://ideas.woocommerce.com/forums/133476-woocommerce?category_id=84283&query=notifications) and I didn't see one asking to mute the push notification for free orders but I did find one [asking to mute notifications during non-working hours](https://ideas.woocommerce.com/forums/133476-woocommerce/suggestions/37906687-ability-to-mute-notifications). I think it would be great to add a new idea (or link to an existing one if I missed it) to keep track of it. I really like the idea of more notifications customization options!
